src/snyk/common/error/errorReporter.ts

```diff
@@ -10,6 +10,11 @@ import { OnUncaughtException } from './integrations/onUncaughtException';
 
 type SentryEnvironment = 'development' | 'preview' | 'production';
 
+export enum TagKeys {
+  CodeRequestId = 'code_request_id',
+}
+export type Tags = { [key in TagKeys]?: string };
```

Comment:
It'd be cool to have types for values of the keys as well, but could be a pain to define them :) See if that's something feasible 😄 nice job anyway 👍

Reply:
Already discussed. 

tldr; Agree. We can always add types as we need to later on